### PR TITLE
ISPN-15286 Upgrade OpenTelemetry to 1.31 and remove okhttp in favour or the JDK lib

### PR DESCRIPTION
**Pull request by fax4ever**
https://issues.redhat.com/browse/ISPN-15286

### CODE REVIEW

**Comment by fax4ever** [comment]
I made a little change, that is not the property `otel.exporter.otlp.protocol` to `http/protobuf` is set by default and document it.

**Comment by domiborges** [comment]
Thank you @fax4ever I added a commit with my suggestions. Let me know if works for you

**Comment by fax4ever** [comment]
@domiborges thank you!

File: documentation/src/main/asciidoc/topics/proc_configuring_tracing.adoc [hunk 1]
```diff
@@ -29,8 +29,20 @@ include::properties/tracing.properties[]
...
+== Tracing data format
+The {brandname} Server, by default, exports tracing data using the OTLP `http/protobuf` protocol.
+
+.tracing.properties
+[source,options="nowrap",subs=attributes+]
+----
+otel.exporter.otlp.protocol = http/protobuf
+----
+
+To use a different protocol, you must manually add more JARS or dependencies to the {brandname} server `lib` directory.
```
Comment (domiborges) [inline documentation/src/main/asciidoc/topics/proc_configuring_tracing.adoc:48]:
```suggestion
To use a different protocol, you must copy  JAR files or dependencies to the `$ISPN_HOME/server/lib` directory of your {brandname} Server installation.
```
One more suggestion :)

Comment (fax4ever) [reply documentation/src/main/asciidoc/topics/proc_configuring_tracing.adoc]: Applied. Thank you

**Comment by fax4ever** [comment]
@domiborges I squashed the last three commits about documentation and I added you as co-author, I don't know if it is fine for you

**Comment by domiborges** [comment]
Great! Thank you for the credit @fax4ever

**Review by pruivo — APPROVED** [review]
LGTM, waiting for CI run to finish.

**Comment by pruivo** [comment]
merged! thanks @fax4ever !

**Comment by fax4ever** [comment]
Thank you @pruivo

**Pull request merged**